packages/trader/src/App/Components/Elements/Errors/__tests__/error-component.spec.jsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import { Router } from 'react-router';
+import { createBrowserHistory } from 'history';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import ErrorComponent from '../error-component';
+
+describe('ErrorComponent', () => {
+    const common_error_message = 'Sorry, an error occured while processing your request.';
+    const test_title = 'Test title';
+    const test_message = 'Test message';
+    const redirect_button = 'Confirm';
+    const default_button = 'Ok';
+    const refresh_button = 'Refresh';
+    const browser_history = createBrowserHistory();
+    const reloadFn = () => {
+        window.location.reload();
+    };
+
+    beforeAll(() => {
+        Object.defineProperty(window, 'location', {
+            configurable: true,
+            value: { reload: jest.fn() },
+        });
+    });
+
+    it('should render ErrorComponent', () => {
+        const { container } = render(
+            <Router history={browser_history}>
+                <ErrorComponent />
+            </Router>
+        );
+
+        expect(container).toBeInTheDocument();
+    });
+    it('should render child component Dialog if ErrorComponent recive is_dialog === true', () => {
+        render(<ErrorComponent is_dialog />);
```

Comment:
```suggestion
        render(<ErrorComponent />);
```
If is_dialog default's value is  `true` then we dont need to be pass is as props.
Just like you did in the `line 45`.

Reply:
Unfortunately `is_dialog` has no default value